src/App/Init.php

```diff
@@ -114,3 +114,17 @@
 // Rate limiter violation handler
 $di->setter['BehEh\Flaps\Flap']['setViolationHandler'] =
 	$di->lazyNew('Ushahidi\App\ThrottlingViolationHandler');
+
+/**
+ * @TODO
+ * This should only be for our csv exporter
+ */
+if (php_sapi_name() !== "cli") {
```

Comment:
@rowasc Does this behave as expected when the cli is called from the API cli route? I'm not sure whether or not that enters a totally separate process.